.npmrc

```diff
@@ -1 +1 @@
-auto-install-peers = true
+legacy-peer-deps=true
```

Comment:
What's requiring this? I think ideally we wouldn't need it.

Reply:
The WordPress dependencies. They're working on fixing it but that fix isn't out yet. It's this problem: https://github.com/WordPress/gutenberg/issues/48009 They're working on replacing `reakit` with `ariakit` but that's not complete yet. When it is, we can stop using legacy peer deps.

Reply:
Actually, apparently `reakit` is now fully removed, but we need to make sure that the versions of the dependencies we are loading for the version of WP that we're using are the ones with reakit removed. Hopefully that'll be true when WP 6.5 releases but we will need to confirm.